ansible_collections/arista/avd/plugins/plugin_utils/eos_validate_state_utils/ansible_eos_device.py

```diff
@@ -17,7 +17,13 @@
 
 logger = getLogger(__name__)
 
+REQUIRED_ANTA_VERSION = "v0.13.0"
```

Comment:
Can we somehow try to read this from some other place to avoid to maintain multiple mentions of this
If not can you add a comment above this saying that this is temporary until ANTA mode is out of preview and the `anta` library version requirement is added to AVD main requirement file and this MUST be removed after?

Reply:
I decided to just put a note in 183de5840e99bdad7fdf14a5aea6e20aa8016765. GA is coming soon so manual work is fine until then.